internal/event/event_observation_test.go

```diff
@@ -503,6 +503,50 @@ func Test_composeFromTelemetryFiltering(t *testing.T) {
 				},
 			},
 		},
+		{
+			name:   "with-request-client-headers-with-telemetry",
+			fromOp: Op("with-request-client-headers-with-telemetry"),
+			opts: []Option{
+				WithId("with-request-client-headers-with-telemetry"),
+				WithRequestInfo(TestRequestInfo(t)),
+				WithFlush(),
+				WithRequest(&Request{
+					Operation: "op",
+					Endpoint:  "/auth-tokens/<id>",
+					Details:   testWorkerStatus(t),
+					UserAgent: &UserAgent{
```

Comment:
Looks like this should be 
```suggestion
					UserAgents: []*UserAgent{{
```

Reply:
Done, thank you!